# Stereo Synchronize Node Parameters

rosbag_folder_path: "/mnt/c/Users/adiee/OneDrive - University of Cape Town/Documents/MASTERS/Experimental Work/Dataset Creation/Ship-Hull-Vinyl-Dataset/Data" #path/to/folder
unsynched_bag_name: "ship_hull_vinyl_raw.bag"
cam0_topic: /video_source_0/raw
cam1_topic: /video_source_1/raw